composer.json

```diff
@@ -15,15 +16,29 @@
 		"squizlabs/php_codesniffer": "^3.6",
 		"wp-coding-standards/wpcs": "^2.3",
 		"yoast/phpunit-polyfills": "^1.0",
-		"thor-juhasz/phpunit-coverage-check": "^0.3.0"
+		"thor-juhasz/phpunit-coverage-check": "^0.3.0",
+		"humbug/php-scoper": "0.17.5"
+	},
+	"autoload": {
+		"classmap": [
+			"vendor_prefixed/"
+		]
 	},
 	"scripts": {
-	  "lint": "phpcs --standard=ruleset.xml",
-	  "test": "phpunit",
-	  "test:install": "bash ./bin/install-wp-tests-local.sh",
-	  "coverage": "composer run coverage:test && composer run coverage:check",
-	  "coverage:test": "phpdbg -qrr vendor/phpunit/phpunit/phpunit --coverage-clover coverage-report/clover.xml",
-	  "coverage:check": "phpunit-coverage-check -t 54 coverage-report/clover.xml"
+		"lint": "phpcs --standard=ruleset.xml",
+		"test": "phpunit",
+		"test:install": "bash ./bin/install-wp-tests-local.sh",
+		"coverage": "composer run coverage:test && composer run coverage:check",
+		"coverage:test": "phpdbg -qrr vendor/phpunit/phpunit/phpunit --coverage-clover coverage-report/clover.xml",
+		"coverage:check": "phpunit-coverage-check -t 54 coverage-report/clover.xml",
+		"post-install-cmd": [
+			"@php ./vendor/bin/php-scoper add-prefix --output-dir=./vendor_prefixed --config=php-scoper/dependency-injection.inc.php --force",
```

Comment:
what does this `@` means? why `--force`?

Reply:
@Nevoss @php ensures that the command is run with the same PHP binary that Composer itself is using. Reference - https://stackoverflow.com/questions/32750250/tell-composer-to-use-different-php-version

Reply:
When the ./vendor/bin/php-scoper add-prefix is executed if it finds vendor_prefixed it will prompt `The output directory .../vendor_prefixed" already exists.` so using --force is to regenerate the folder without the prompt